README.md

```diff
@@ -22,3 +22,16 @@ This will install the project
 - `./manage.py runserver`
 
 Now navigate to [localhost:8000](http://localhost:8000)
+
+## App dependency graph (not strictly enforced)
```

Comment:
Well in an ideal world it might be nice for maintainability to adhere to a dependency graph such as this one :grinning:. It's now not the case however, the `UserDetails` app has references to `CreditManagement` and `Dining`, and `CreditManagement` has references to `Dining`. So this particular graph is not correct/enforced in this case but I'm trying to adhere to it more and more.

Reply:
If you explain the intent like that it is clear to me, good initiative! Maybe you can put part of your comment above in the comments, so it is clear to all readers what you mean with 'not strictly enforced'